vnext/Scripts/Microsoft.ReactNative.targets

```diff
@@ -5,8 +5,8 @@
   <PropertyGroup>
     <Native-Platform Condition="'$(Platform)' == 'Win32'">x86</Native-Platform>
     <Native-Platform Condition="'$(Platform)' != 'Win32'">$(Platform)</Native-Platform>
-    <_rnwDebugFolder>$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(Native-Platform)\native\debug\</_rnwDebugFolder>
-    <_rnwReleaseFolder>$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(Native-Platform)\native\release\</_rnwReleaseFolder>
+    <_rnwDebugFolder>$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(Native-Platform)\native-debug\</_rnwDebugFolder>
```

Comment:
Why `native-$(ConfigurationLowerCase)` instead of `native\$(Configuration)`?

Reply:
@JunielKatarn : I'll use the variable. but the nuget convention is to include all files under native\**.dll. See https://docs.microsoft.com/en-us/nuget/guides/create-uwp-packages#adding-the-native-implementation-libraries